### PR TITLE
Port publiccode.yml to release core-0.2;it-0.2

### PR DESCRIPTION
**Pull request by libremente**
This PR ports the current version of the `publiccode.yml` file to the [latest release](https://docs.italia.it/italia/developers-italia/publiccodeyml/it/master/index.html) of the standard. 
As such, some keys are changed, others have been removed (see the changelog [here](https://github.com/italia/publiccode.yml/blob/master/CHANGELOG.rst)).

Note: I inserted the `codiceIPA: unical` key since the `repoOwner` is `Unical`.

Please also note that this is a WIP PR and, as such, it needs further improvements in order to be ready to merge. As such, I've inserted some comments in the file which should be reviewed by members of this project in order to be solved.
 
Thank you very much for your efforts! :+1:  @peppelinux @francesco-filicetti

### CODE REVIEW

File: publiccode.yml [hunk 1]
```diff
@@ -21,113 +15,67 @@ outputTypes:
...
+  en:
     localisedName: Peo
     genericName: Django peo
-    shortDescription: >
+    shortDescription: |- 
           Gestione, redazione e pubblicazione Bandi e Concorsi per
           personale Tecnico Amministrativo e personale Docente.
 
-    longDescription: >
+    longDescription: |-
```
Comment (libremente) [inline publiccode.yml:36]: This key does not pass validation. The standard mandates to have a 
`Tipo: stringa (min 500 caratteri, max 10000 caratteri)` so some more chars are needed.

File: publiccode.yml [hunk 2]
```diff
@@ -21,113 +15,67 @@ outputTypes:
...
   authorsFile: AUTHORS
 
 maintenance:
-  type: "contract"
-
-  contractors:
-    - name: "Fornitore Privato SPA"
-      website: "https://privatecompany.com"
-      until: "2019-01-01"
+  type: "internal"
```
Comment (libremente) [inline publiccode.yml:59]: Is this correct?

File: publiccode.yml [hunk 3]
```diff
@@ -21,113 +15,67 @@ outputTypes:
...
 
   contacts:
```
Comment (libremente) [inline publiccode.yml:61]: Seeing the standard, this should be `Questa chiave contiene il nome completo di uno dei contatti tecnici. Deve essere una persona reale; NON popolare questa chiave con informazioni di contatto generiche, dipartimenti dell’azienda, associazioni, etc.` https://docs.italia.it/italia/developers-italia/publiccodeyml/it/master/schema.core.html#contatto

File: publiccode.yml [hunk 4]
```diff
@@ -21,113 +15,67 @@ outputTypes:
...
   authorsFile: AUTHORS
 
 maintenance:
-  type: "contract"
-
-  contractors:
-    - name: "Fornitore Privato SPA"
-      website: "https://privatecompany.com"
-      until: "2019-01-01"
+  type: "internal"
```
Comment (libremente) [inline publiccode.yml:60]: Removed since it does not seem right.

File: publiccode.yml [hunk 5]
```diff
@@ -21,113 +15,67 @@ outputTypes:
...
           Gestione, redazione e pubblicazione Bandi e Concorsi per
           personale Tecnico Amministrativo e personale Docente.
 
-    longDescription: >
+    longDescription: |- 
           Gestione, redazione e pubblicazione Bandi e Concorsi per
           personale Tecnico Amministrativo e personale Docente.
 
     documentation: "https://peo.unical.it/docs"
-    apiDocumentation: "https://docs.djangoproject.com/en/2.1/"
```
Comment (libremente) [inline publiccode.yml:54]: Removed since it is not specific to the project itself.

File: publiccode.yml [hunk 6]
```diff
@@ -21,113 +15,67 @@ outputTypes:
 platforms:
   - GNU/Linux
```
Comment (libremente) [inline publiccode.yml:16]: Is this right? Seeing the standard, this key should represent: `Questa chiave specifica su quale piattaforma funziona il software. È pensata per descrivere le piattaforme che l’utente userà per accedere ed utilizzare il software, piuttosto che la piattaforma sul quale il software gira.`

File: publiccode.yml [hunk 7]
```diff
@@ -1,17 +1,11 @@
...
 name: Peo
-url: "https://github.com/italia/django-peo.git"
-landingURL: "https://github.com/italia/django-peo"
+url: "https://github.com/UniversitaDellaCalabria/PEO.git"
+landingURL: "https://github.com/UniversitaDellaCalabria/PEO"
 
-#isBasedOn: "https://github.com/italia/otello.git"
 softwareVersion: "1.0"
 releaseDate: "2019-02-15"
-#logo: img/logo.svg
```
Comment (libremente) [inline publiccode.yml:13]: Do you have a logo? It could be nice! :+1:

**Comment by peppelinux** [comment]
Hi I made a brand new redaction of publiccode.yml using [public code editor italia](https://publiccode-editor.developers.italia.it/#), thank you for the contribution. Hope to made it well

**Pull request closed without merging**